File: Application/dotNET/DataLayer/Models/Voyage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataLayer.Models
{
    public class Voyage
    {
        public int? voyage_id { get; set; }
        public int? tanker_id { get; set; }
        public string? voyage_status { get; set; }
        public DateTime? start_time_utc { get; set; }
        public DateTime? end_time_utc { get; set; }
        public int? start_position_id { get; set; }
        public int? end_position_id { get; set; }
        public string? start_port_name { get; set; }
        public string? end_port_name { get; set; }
        public string? destination_final {  get; set; }
        public DateTime? eta_final { get; set; }
        public DateTime created_at { get; set; }
        public DateTime updated_at { get; set; }
    }
}

File: Application/React_Frontend/src/types/Voyage.ts
export interface Voyage {
  Voyage_Id: number | null;
  Tanker_Id: number | null;

  Voyage_status: string | null;

  Start_Time_Utc: string | null;
  End_Time_Utc: string | null;

  Start_Position_Id: number | null;
  End_Position_Id: number | null;

  Start_Port_Name: string | null;
  End_Port_Name: string | null;

  Destination_Final: string | null;

  Eta_final: string | null;

  Reated_At: string;
  Updated_At: string;
}